lib/apiservers/portlayer/swagger.yml

```diff
@@ -193,6 +193,56 @@ paths:
           description: "error"
           schema:
             $ref: "#/definitions/Error"
+  /storage/volumes/{name}:
+    get:
+      description: "Get a Volume Handle"
+      operationId: GetVolume
+      tags: ["volumes"]
+      consumes:
+        - application/json
+        - application/octet-stream
+      produces:
+        - application/json
+      parameters:
+        - name: name
+          required: true
+          in: path
+          type: string
+      responses:
+        '404':
+          description: "Volume not found"
+          schema:
+            $ref: "#/definitions/Error"
+        '200':
+          description: "OK"
+          schema:
+            type: string
+        default:
+          description: "Error"
+          schema:
+            $ref: "#/definitions/Error"
```

Comment:
Added an entry for get. without a stub. this is a placeholder should we actually want a publicly exposed get for the volumes.